data/randomization/randomization-with-weather.ipynb

```diff
@@ -0,0 +1,732 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.        if len(normal_weather_data) &gt;=2:
</pre><p>Seems that this is a check to make sure there are enough file from a particular site and month? Maybe add a comment about this to make sure it is clear.</p><p> </p><p>It is probably a good idea for you to be able to easily see which site-month combinations have or not have sampled files, as a separate dataframe.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/uw-echospace/bat-detector-validation/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>